proxy/src/main/java/com/velocitypowered/proxy/connection/backend/VelocityServerConnection.java

```diff
@@ -133,6 +137,24 @@ private String createLegacyForwardingAddress() {
     return data.toString();
   }
 
+  private String createBungeeGuardForwardingAddress(byte[] forwardingSecret) {
+    StringBuilder data = new StringBuilder()
```

Comment:
Instead of duplicating `createLegacyForwardingAddress()`, perhaps we can make this more generic by refactoring `createLegacyForwardingAddress()` to allow the list of profile properties to vary.